pkg/datastore/mysql/query_test.go

```diff
@@ -262,6 +262,20 @@ func TestBuildFindQuery(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "query with IN operator",
+			kind: "Project",
+			listOptions: datastore.ListOptions{
+				Filters: []datastore.ListFilter{
+					{
+						Field:    "Status",
+						Operator: "in",
+						Value:    []int32{1, 2, 3},
+					},
+				},
+			},
+			expectedQuery: "SELECT Data FROM Project WHERE Status IN (?,?,?)",
+		},
```

Comment:
It would be nice if we have the case `Value` slice containing only one element.

Reply:
Sure 👍 Addressed on https://github.com/pipe-cd/pipe/pull/1766/commits/c51946ecbc852a77b88f97dc1655c552f375b206 🙏